script/api.py

```diff
@@ -0,0 +1,110 @@
+import os
+
+import psycopg2
+from flask import Flask, jsonify, request, session
+
+app = Flask(__name__)
+
+hostname = os.getenv("POSTGRES_HOST")
+database = os.getenv("POSTGRES_DB")
+postgres_user = os.getenv("POSTGRES_USER")
+port_id = os.getenv("POSTGRES_PORT")
+
+conn = psycopg2.connect(
+    host=hostname, dbname=database, user=postgres_user, port=port_id
+)    
+
+cur = conn.cursor()
+
+
+@app.route("/signup", methods=["POST"])
+def signup():
+    """SIGNUP API
+    Parameters:
+    username (String): Username of user
+    email (String): Email of user
+    password (String): Password of user
+
+    Returns:
+        Http Response: {
+            String : Message
+            Integer : Status Code
+        }
+    """
+
+    username = request.json.get("username")
+    email = request.json.get("email")
+    password = request.json.get("password")
```

Comment:
Add validations to the request body.
For example, we need to check whether the username, email, and password are not null in the body.
Also, their type should be as per the request need. Here all the fields must be a string.

In case any of the required fields is missing or is not of the correct type, we return an error message stating which field is missing with a 400 error code.